src/renderer/src/routes/(MapTabs)/_Map.tab1.tsx

```diff
@@ -4,10 +4,10 @@ import { createFileRoute } from '@tanstack/react-router'
 import { Text } from '../../components/Text'
 
 export const Route = createFileRoute('/(MapTabs)/_Map/tab1')({
-	component: RouteComponent,
+	component: Observations,
```

Comment:
this file name is very confusing. Can we name it Map.Observations.tsx or Map.tabobservations.tsx or something?

Reply:
these file names and components are going to change. These are temporary and so I don't think it is necessary to change right now